concourse/pipeline/job_def.lib.yml

```diff
@@ -29,6 +29,14 @@ res_plpython3_bin: bin_plpython3_gpdb6_ubuntu18
 os: ubuntu1804
 #@ end
 
+#! Job config for gpdb7 rocky8
```

Comment:
since we have both gpdb6 and gpdb7, please modify the `build_test_job_name` so the job name can show the gpdb version. It is a bit confusing otherwise.